src/components/Toast.js

```diff
@@ -302,7 +304,7 @@ class Toast extends Component {
           {closeButton && closeButton}
           {autoClose && (
```

Comment:
If `autoClose` is set to false the progress bar is not displayed and using the `progress` props will have no effect. 

I think about the following scenario.
1- user display a toast without duration saying: `Your upload will start shortly`
2- upload start and the user want to display the progress, you get the idea

 ```suggestion
          {(autoClose || progress) && (
```